src/HolographOperator.sol

```diff
@@ -836,7 +841,7 @@ contract HolographOperator is Admin, Initializable, HolographOperatorInterface {
       /**
        * @dev check if smart contract is owned by sender
        */
-      require(Ownable(operator).isOwner(msg.sender), "HOLOGRAPH: sender not owner");
+      require(Ownable(operator).owner() == msg.sender, "HOLOGRAPH: sender not owner");
```

Comment:
Just curious what the difference is between these two functions is for future reference

Reply:
For ownable type, owner function is default. IsOwner is not in spec and will not always be present. So we went more strict and are using a function that is by the spec.

Reply:
ah okay cool